src/main/java/core/basesyntax/FileWork.java

```diff
@@ -9,6 +15,25 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        final char W = 'w';
+        ArrayList<String> wordsWithWList = new ArrayList<String>();
+        String text = "";
+        try {
+            text = new String(Files.readAllBytes((Paths.get(fileName)))).toLowerCase().replaceAll(
+                    "[^A-Za-zА-Яа-я ]", "");
```

Comment:
your solution has one drawback:
the last word on the line is sticked together with the first word on the next line

`word1 word2
word 3` 
=> `word1 word2word3`

let's try retrieving lines first, and then split each line